source/agora/node/Ledger.d

```diff
@@ -42,11 +44,16 @@ public class Ledger
     /// Temporary storage where transactions are stored until blocks are created.
     private Transaction[] storage;
 
+    /// ledger data dir
+    private string data_dir;
+
     /// Ctor
-    public this ()
+    public this (const string data_dir = ".cache")
```

Comment:
There shouldn't be a default value here.

Reply:
delete it

Reply:
In many unittest, I will put the default value in the part where I create the bedget.